include/tester/db/Database.hpp

```diff
@@ -76,8 +48,9 @@ class Database
 
     bsoncxx::stdx::optional<std::string> getField(std::string collectionName, std::string id, std::string field_name)
```

Comment:
`field_name` passed by value and is being copied multiple times. use `const &`

Reply:
done